alternator/executor.cc

```diff
@@ -895,9 +923,20 @@ static void verify_billing_mode(const rjson::value& request) {
         if (!rjson::find(request, "ProvisionedThroughput")) {
             throw api_error::validation("When BillingMode=PROVISIONED, ProvisionedThroughput must be specified.");
         }
+        const rjson::value& throughput = request["ProvisionedThroughput"];
+        auto rcu = get_int_attribute(throughput, "ReadCapacityUnits");
+        auto wcu = get_int_attribute(throughput, "WriteCapacityUnits");
+        if (!rcu.has_value()) {
+            throw api_error::validation("validation error detected: Value null at 'provisionedThroughput.readCapacityUnits' failed to satisfy constraint: Member must not be null");
```

Comment:
nitpick: the error message doesn't need to begin with "validation error detected", it's already in the "validation error" type and will be printed as such.
And why is this message so complicated? Why not just say that ReadCapacityUnits is missing (not "null")? Did you copy this string from DynamoDB or something?

Reply:
I copy it from DynamoDB